# datasource exception messages
exception.datasource.connection=Error connecting to the datasource

# order exception messages
exception.order.save=Error saving order
exception.order.find=Error retrieving order
exception.order.process=Error processing order

# customer exception messages
exception.customer.find.by.id=Error retrieving customer by id
exception.customer.find.by.company=Error retrieving customer by company name
exception.customer.save=Error saving customer